src/phrases/phrases-nn.ts

```diff
@@ -1115,6 +1115,10 @@ const phrases = {
         label: "Skriv inn kommentar",
       },
       updateComment: "Oppdater kommentar",
+      deleteCurrentRevision: {
+        title: "Slett gjeldande revisjon",
+        dialogText: "Er du sikker på at du vil slette gjeldane revisjon? Det er ikkje mogleg å gjenopprette han.",
+      },
     },
     status: {
       new: "Ny - Ikke lagret",
```

Comment:
Ikkje egentlig relevant...
```suggestion
      new: "Ny - Ikkje lagra",
```